x-pack/test/saved_object_api_integration/common/suites/resolve.ts

```diff
@@ -71,7 +71,7 @@ export const TEST_CASES = Object.freeze({
 });
 
 export function resolveTestSuiteFactory(esArchiver: any, supertest: SuperTest<any>) {
-  const expectSavedObjectForbidden = expectResponses.forbiddenTypes('get');
+  const expectSavedObjectForbidden = expectResponses.forbiddenTypes('bulk_get');
```

Comment:
The Secure SOC wrapper used to check privileges differently for `SOC.resolve` and `SOC.bulkResolve` (it checked 'get' and 'bulk_get' privilege actions, respectively).

Now, the `internalBulkResolve` module is initiating authorization checks directly using the security extension, but it is used for the implementation of both the `SOC.resolve` and `SOC.bulkResolve` functions.

The cleanest change was to make `internalBulkResolve` always check for the 'bulk_get' privilege action, and change the `resolve` integration test to reflect that.